### PR TITLE
DOC: Adds warning banner if not browsing the latest version of the docs

### PR DESCRIPTION
**Pull request by gethvi**
As promised. :sunglasses: 

Also fixes IntelMQ logo showing correctly in the repo as well as in the docs.

### CODE REVIEW

File: docs/_overrides/main.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+{#SPDX-License-Identifier: AGPL-3.0-or-later#}
+
+{% extends "base.html" %}
+
+{% block outdated %}
+  You're not viewing the latest version.
+  <a href="{{ '../' ~ base_url }}" style="color: black">
+    <strong>Click here to go to latest.</strong>
+  </a>
+{% endblock %}
```
Comment (sebix) [inline docs/_overrides/main.html]: missing newline

Comment (gethvi) [reply docs/_overrides/main.html]: Thanks. Fixed.

**Comment by gethvi** [comment]
It seems to break the top bar of the page. :-( Will look into it.

**Review by kamil-certat — APPROVED** [review]

**Comment by aaronkaplan** [comment]
thx a lot @gethvi ! merging.

**Pull request merged**